src/SAWScript/HeapsterBuiltins.hs

```diff
@@ -1020,7 +1022,7 @@ heapster_assume_fun_rename _bic _opts henv nm nm_to perms_string term_string =
   do Some lm <- failOnNothing ("Could not find symbol: " ++ nm)
                               (lookupModContainingSym henv nm)
      sc <- getSharedContext
-     let w = llvmModuleArchReprWidth lm
+     let w = llvmModuleArchReprWidth lm -- Should hardcode 64
```

Comment:
This shouldn't actually be hard coded to 64, since it could potentially be useful to have it match some other architecture with a different bit width. Currently, the way to achieve that is to specify it in the Heapster environment. I could imagine making this an argument instead of requiring a Heapster environment, but the current approach seems easiest for now, so maybe just remove this comment?